tests/src/Integration/RulesEntityIntegrationTestBase.php

```diff
@@ -53,15 +53,47 @@ public function setUp() {
     $this->languageManager->getCurrentLanguage()->willReturn($language->reveal());
     $this->languageManager->getLanguages()->willReturn([$language->reveal()]);
 
-    $entityType = new ContentEntityType([
-      'id' => 'test',
-      'label' => 'Test',
-      'entity_keys' => [
-        'bundle' => 'bundle',
+    // We need to support multiple entity types, including a test type:
+    $type_info = [
+      'test' => [
+        'id' => 'test',
+        'label' => 'Test',
+        'entity_keys' => [
+          'bundle' => 'bundle',
+        ],
       ],
-    ]);
+      'user' => [
+        'id' => 'user',
+        'label' => 'Test User',
+        'entity_keys' => [
+          'bundle' => 'user',
+        ],
+      ],
+      'user_role' => [
+        'id' => 'user_role',
+        'label' => 'Test Role',
+        'entity_keys' => [
+          'bundle' => 'user_role',
+        ],
+      ],
+      'node' => [
+        'id' => 'node',
+        'label' => 'Test Node',
+        'entity_keys' => [
+          'bundle' => 'dummy',
+        ],
+      ],
+    ];
+
+    $type_array = [];
+
+    foreach ($type_info as $type => $info) {
+      $entity_type = new ContentEntityType($info);
```

Comment:
We do need a mock for it, so I'll make sure that it's passed as a ConfigEntityType.